tests/lib.rs

```diff
@@ -59,8 +59,10 @@ fn unify_activated() {
     }
 
     fn also_maybe(a: &mut Capture<dyn Activated>) {
-        a.filter("whatever filter string, this won't be run anyway", false)
+        let filter = a
+            .compile_filter("whatever filter string, this won't be run anyway", false)
```

Comment:
what?

Reply:
PCAP_ERROR_BREAK if the loop terminated due to a call to pcap_breakloop() before any packets were processed.